Linter issues of Golang CI linters pack

**Describe the bug**
We have a lot of linter issues related with new version of golangci-linter

**To Reproduce**
Steps to reproduce the behavior:
1. Run `make lint`
2. See the errors as result:

```sh
+ lint
pkg/input/inquirer.go:97:10: appendAssign: append result not assigned to the same slice (gocritic)
			dst = append(src[:i], src[i+1:]...)
			      ^
pkg/commands/api.go:34:1: don't use `init` function (gochecknoinits)
func init() {
^
pkg/commands/gke.go:26:1: don't use `init` function (gochecknoinits)
func init() {
^
pkg/commands/root.go:59:1: don't use `init` function (gochecknoinits)
func init() {
^
pkg/commands/api.go:14:1: `apiCmd` is a global variable (gochecknoglobals)
var apiCmd = &cobra.Command{
^
pkg/commands/gke.go:14:1: `gkeCmd` is a global variable (gochecknoglobals)
var gkeCmd = &cobra.Command{
^
pkg/commands/root.go:16:1: `cfgFile` is a global variable (gochecknoglobals)
var cfgFile string
^
pkg/commands/root.go:19:1: `RootCmd` is a global variable (gochecknoglobals)
var RootCmd = &cobra.Command{
^
pkg/commands/driver.go:13:1: `databasePort` is a global variable (gochecknoglobals)
var (
^
pkg/commands/driver.go:19:1: `driverCmd` is a global variable (gochecknoglobals)
var driverCmd = &cobra.Command{
^
pkg/version/version.go:3:1: `RELEASE` is a global variable (gochecknoglobals)
var (
^
pkg/commands/config.go:13:1: `configCmd` is a global variable (gochecknoglobals)
var configCmd = &cobra.Command{
^
pkg/commands/storage.go:15:1: `storageCmd` is a global variable (gochecknoglobals)
var storageCmd = &cobra.Command{
^
pkg/commands/new.go:17:1: `newCmd` is a global variable (gochecknoglobals)
var newCmd = &cobra.Command{
^
make: *** [lint] Error 1
```

**Expected behavior**
A clear list of issues

**Desktop or Notebook (please complete the following information):**
 - Device: MacBookPro
 - OS: Mac OS 10.14.1 Mojave
 - Go version: 1.11.2

--- a/.templates/base/.helm/templates/configmap.yaml
+++ b/.templates/base/.helm/templates/configmap.yaml
@@ -25,6 +25,7 @@ data:
         "port": {{ .Values.service.info.port }},
         "statistics": {{ .Values.service.info.statistics }}
       },
+      {{[- if .Storage.Enabled ]}}
       {{- if .Values.service.database.enabled }}
       "database": {
         "driver": "{{ .Values.service.database.driver }}",
@@ -51,6 +52,7 @@ data:
         "dir": "migrations",
         "active": true
       },
+      {{[- end ]}}
       "logger": {
         "format": "{{ .Values.service.logger.format }}",
         "level": {{ .Values.service.logger.level }},

--- a/.templates/base/.helm/templates/deployment.yaml
+++ b/.templates/base/.helm/templates/deployment.yaml
@@ -46,6 +46,7 @@ spec:
         env:
         - name: {{ printf "%s_CONFIG_PATH" .Values.service.name | upper | replace "-" "_" }}
           value: /etc/{{ .Values.service.name | replace "-" "_" }}/default.conf
+        {{[- if .Storage.Enabled ]}}
         {{- if .Values.service.database.enabled }}
         {{- if .Values.service.database.name }}
         - name: {{ printf "%s_DB_NAME" $serviceName | upper | replace "-" "_" }}
@@ -69,6 +70,7 @@ spec:
               name: {{ .Values.service.database.password }}
         {{- end }}
         {{- end }}
+        {{[- end ]}}
         {{- if .Values.livenessProbe.enabled }}
         livenessProbe:
           httpGet:
@@ -93,7 +95,7 @@ spec:
           mountPath: /etc/{{ .Values.service.name | replace "-" "_" }}/
         resources:
 {{ toYaml .Values.resources | indent 12 }}
-     volumes:
+      volumes:
         - name: {{ template "name" . }}-config
           configMap:
             name: {{ template "name" . }}-config

--- a/.templates/base/.helm/values-dev.yaml
+++ b/.templates/base/.helm/values-dev.yaml
@@ -160,6 +160,8 @@ service:
     ##
     statistics: true
 
+  {{[- if .Storage.Enabled ]}}
+
   ## Database attributes
   ##
   database:
@@ -200,6 +202,7 @@ service:
     connections:
       max: {{[ .Storage.Config.Connections.Max ]}}
       idle: {{[ .Storage.Config.Connections.Idle ]}}
+  {{[- end ]}}
 
   ## Logger format and level
   ##

--- a/.templates/base/.helm/values-prod.yaml
+++ b/.templates/base/.helm/values-prod.yaml
@@ -160,6 +160,8 @@ service:
     ##
     statistics: true
 
+  {{[- if .Storage.Enabled ]}}
+
   ## Database attributes
   ##
   database:
@@ -200,6 +202,7 @@ service:
     connections:
       max: {{[ .Storage.Config.Connections.Max ]}}
       idle: {{[ .Storage.Config.Connections.Idle ]}}
+  {{[- end ]}}
 
   ## Logger format and level
   ##

--- a/.templates/base/Makefile
+++ b/.templates/base/Makefile
@@ -115,7 +115,7 @@ endif
 
 cluster:
 	@echo "+ $@"
-ifneq ("$(GKE_CLUSTER_NAME)", "$(shell gcloud container clusters list | grep $(GKE_CLUSTER_NAME) | cut -d ' ' -f1)")
+ifneq ("$(KUBE_CONTEXT)", "$(shell kubectl config get-clusters | grep $(KUBE_CONTEXT))")
 	@gcloud container clusters get-credentials $(GKE_CLUSTER_NAME) --zone $(GKE_PROJECT_ZONE) --project $(GKE_PROJECT_ID)
 endif
 ifneq ("$(KUBE_CONTEXT)", "$(shell kubectl config current-context)")
@@ -238,11 +238,13 @@ run: clean build {{[- if .Storage.Enabled ]}} db {{[- end ]}}
 		-e "{{[ toENV .Name ]}}_INFO_PORT=$({{[ toENV .Name ]}}_INFO_PORT)" \
 		-e "{{[ toENV .Name ]}}_LOGGER_LEVEL=$({{[ toENV .Name ]}}_LOGGER_LEVEL)" \
 		-e "{{[ toENV .Name ]}}_CONFIG_PATH=$({{[ toENV .Name ]}}_CONFIG_PATH)" \
+{{[- if .Storage.Enabled ]}}
 		-e "{{[ toENV .Name ]}}_DB_DRIVER=$({{[ toENV .Name ]}}_DB_DRIVER)" \
 		-e "{{[ toENV .Name ]}}_DB_HOST=$({{[ toENV .Name ]}}_DB_HOST)" \
 		-e "{{[ toENV .Name ]}}_DB_PORT=$({{[ toENV .Name ]}}_DB_PORT)" \
-		-v $(shell pwd)/config/default.conf:$({{[ toENV .Name ]}}_CONFIG_PATH):ro \
 		-v $(shell pwd)/fixtures:/$({{[ toENV .Name ]}}_DB_FIXTURES_DIR):ro \
+{{[- end ]}}
+		-v $(shell pwd)/config/default.conf:$({{[ toENV .Name ]}}_CONFIG_PATH):ro \
 		-d $(CONTAINER_IMAGE):$(BUILD)
 	@sleep 1
 	@docker logs $(CONTAINER_NAME)
@@ -253,12 +255,12 @@ logs:
 
 deploy: push cluster
 	@echo "+ $@"
-	@helm upgrade $(CONTAINER_NAME) -f .helm/$(VALUES).yaml .helm --kube-context $(KUBE_CONTEXT) \
+	@helm upgrade $(APP)-$(NAMESPACE) -f .helm/$(VALUES).yaml .helm --kube-context $(KUBE_CONTEXT) \
 		--namespace $(NAMESPACE) --version=$(RELEASE) --set image.tag=$(BUILD) -i --wait
 
 charts:
 	@echo "+ $@"
-	@helm template .helm -n ${NAMESPACE} -f .helm/${VALUES}.yaml
+	@helm template .helm -n $(APP)-$(NAMESPACE) --namespace $(NAMESPACE) -f .helm/$(VALUES).yaml
 
 test:
 	@echo "+ $@"

--- a/.templates/base/pkg/commands/root.go
+++ b/.templates/base/pkg/commands/root.go
@@ -1,3 +1,6 @@
+// Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/.templates/base/pkg/commands/serve.go
+++ b/.templates/base/pkg/commands/serve.go
@@ -1,3 +1,6 @@
+// Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/.templates/base/pkg/system/signal.go
+++ b/.templates/base/pkg/system/signal.go
@@ -163,6 +163,7 @@ func isSignalAvailable(signal os.Signal, list []os.Signal) bool {
 func removeSignal(signal os.Signal, list []os.Signal) (signals []os.Signal) {
 	for ind, sig := range list {
 		if sig == signal {
+			// nolint: gocritic
 			signals = append(list[:ind], list[ind+1:]...)
 			return
 		}

--- a/.templates/base/pkg/version/version.go
+++ b/.templates/base/pkg/version/version.go
@@ -1,3 +1,5 @@
+// Package version contains global variables for
+// nolint: gochecknoglobals, gochecknoinits
 package version
 
 var (

--- a/.templates/storage/base/pkg/commands/down.go
+++ b/.templates/storage/base/pkg/commands/down.go
@@ -1,3 +1,6 @@
+// Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/.templates/storage/base/pkg/commands/migrate.go
+++ b/.templates/storage/base/pkg/commands/migrate.go
@@ -1,3 +1,6 @@
+// Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/.templates/storage/base/pkg/commands/up.go
+++ b/.templates/storage/base/pkg/commands/up.go
@@ -1,3 +1,6 @@
+// Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/pkg/commands/api.go
+++ b/pkg/commands/api.go
@@ -1,4 +1,6 @@
 // Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/pkg/commands/config.go
+++ b/pkg/commands/config.go
@@ -1,3 +1,6 @@
+// Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/pkg/commands/driver.go
+++ b/pkg/commands/driver.go
@@ -1,3 +1,6 @@
+// Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/pkg/commands/gke.go
+++ b/pkg/commands/gke.go
@@ -1,4 +1,6 @@
 // Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/pkg/commands/new.go
+++ b/pkg/commands/new.go
@@ -1,3 +1,6 @@
+// Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/pkg/commands/root.go
+++ b/pkg/commands/root.go
@@ -1,3 +1,6 @@
+// Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/pkg/commands/storage.go
+++ b/pkg/commands/storage.go
@@ -1,4 +1,6 @@
 // Package commands process flags/environment variables/config file
+// It contains global variables with configs and commands
+// nolint: gochecknoglobals, gochecknoinits
 package commands
 
 import (

--- a/pkg/input/inquirer.go
+++ b/pkg/input/inquirer.go
@@ -94,6 +94,7 @@ func Inquire(cfg *config.Config) *config.Config {
 func delete(src []string, value string) (dst []string) {
 	for i, v := range src {
 		if v == value {
+			// nolint: gocritic
 			dst = append(src[:i], src[i+1:]...)
 		}
 	}

--- a/pkg/version/version.go
+++ b/pkg/version/version.go
@@ -1,3 +1,5 @@
+// Package version contains global variables for
+// nolint: gochecknoglobals, gochecknoinits
 package version
 
 var (

